src/test/util_string_tests.cpp

```diff
@@ -83,4 +84,12 @@ BOOST_AUTO_TEST_CASE(ConstevalFormatString_NumSpec)
     FailFmtWithError<1>("%1$", err_term);
 }
 
+BOOST_AUTO_TEST_CASE(tinyformat_try_format)
+{
+    // ensure invalid format strings don't throw at run-time
```

Comment:
comment seems redundant here

Reply:
I don't agree it's redundant:
- it's not obvious from the tests that we're not just checking error messages, but that we're ensuring no errors are thrown, which is the difference with `tfm::format()`
- additional tests to try_format may be added in the future that test other aspects, so this acts as a little delineator

Reply:
thanks for clarifying